x-pack/platform/plugins/shared/fleet/server/services/epm/packages/get_template_inputs.ts

```diff
@@ -170,12 +170,15 @@ export async function getTemplateInputs(
     savedObjectsClient: soClient,
     ignoreUnverified,
   });
+  // placeholderId is used in case of an otelcol type input and ignored in all other cases
+  const placeHolderId = `<POLICY_ID>`;
```

Comment:
This is needed for the `Configs` tab, otherwise it doesn't append any suffix:

<img width="1759" height="1006" alt="Screenshot 2025-07-29 at 11 24 50" src="https://github.com/user-attachments/assets/ac40b7ed-9d1d-44f4-a4bf-d0b7eb534dcd" />

Reply:
Maybe the suffix is not needed in this view, as this is intended as a basis for the user to compose their own configs, and they will need to take care of configuring their pipelines without conflicts.
Also, this config should only show the OTel parts at the root level, not under `inputs`.

Reply:
ah yes you're right about the `inputs`, I'll fix it

About the placeholder, it depends if the elastic agent needs it or not to work properly.